--- a/src/Runner/TestHandler.php
+++ b/src/Runner/TestHandler.php
@@ -73,7 +73,7 @@ class TestHandler
 	{
 		list($annotations, $testName) = $this->getAnnotations($job->getFile());
 		$testName .= $job->getArguments()
-			? ' [' . implode(' ', preg_replace(array('#["\'-]*(.+?)["\']?$#A', '#(.{30}).+#A'), array('$1', '$1...'), $job->getArguments())) . ']'
+			? ' [' . implode(' ', preg_replace(array('#["\'-]*(.+?)["\']?$#A'), array('$1'), $job->getArguments())) . ']'
 			: '';
 		$annotations += array(
 			'exitcode' => Job::CODE_OK,
